realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -665,19 +698,23 @@ public static String getTableName() {
                 }
             }
         }
+        if (json.has("parentObjects")) {
+            throw new io.realm.exceptions.RealmException("\"@LinkingObject\" field \"parentObjects\" cannot be loaded from json");
```

Comment:
That seems like a reasonable approach, if you prefer.  I'm of the "don't silently ignore stuff" school, so this was my first choice.  Happy to change it, though, if that is more in line with the Realm Way.